src/main/java/com/solvd/pages/mobile/pim/PimPageMobile.java

```diff
@@ -3,27 +3,35 @@
 import com.solvd.mapper.EmployeeMapper;
 import com.solvd.model.Employee;
 import com.solvd.pages.common.pim.PimPageBase;
-import com.zebrunner.carina.utils.factory.DeviceType;
 import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 import java.util.List;
 
-@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = PimPageBase.class)
-public class PimPageMobile extends PimPageBase {
 
-    @FindBy(xpath = "//div[@class = 'oxd-table-filter-header-options']//button")
-    private ExtendedWebElement infoButton;
+public abstract class PimPageMobile extends PimPageBase {
+
+    public PimPageMobile(WebDriver webDriver) {
+        super(webDriver);
+    }
 
     @FindBy(xpath = "//div[@class = 'oxd-table-filter-header']//button")
```

Comment:
I think it would be better to use contains, when checking if element have some class. That way you locator will be more stable, and will work even if some other classes will be added to the element

Reply:
Done